src/scripts/CMSSync/index.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs'
+import api from '../api'
```

Comment:
Maybe rename this in `get` (as it performs an http get) , api is too generic and a bit confusing